audio_analysis.go

```diff
@@ -41,21 +36,24 @@ type AudioAnalysisMeta struct {
 	InputProcess    string  `json:"input_process"`
 }
 
+// A Section represents a large variation in rhythm or timbre, e.g. chorus, verse, bridge, guitar solo, etc.
```

Comment:
`go lint` should warn you about this, but these doc comments should start with the name of the type they are documenting.

For example, this could say:

> Section represents a large variation...

Sorry if it seems I'm being overly pedantic, but I'd prefer not to introduce lint warnings if we can avoid it :-)